pages/configuration/elasticsearch.rst

```diff
@@ -8,108 +8,77 @@ We strongly recommend to use a dedicated Elasticsearch cluster for your Graylog
+| ``elasticsearch_idle_timeout``                     | Duration  | Timeout after which idle connections are terminated          | ``-1s`` (Never)             |
++----------------------------------------------------+-----------+--------------------------------------------------------------+-----------------------------+
+| ``elasticsearch_max_total_connections``            | int       | Maximum number of total Elasticsearch connections            | ``20``                      |
++----------------------------------------------------+-----------+--------------------------------------------------------------+-----------------------------+
+| ``elasticsearch_max_total_connections_per_route``  | int       | Maximum number of Elasticsearch connections per route/host   | ``2``                       |
++----------------------------------------------------+-----------+--------------------------------------------------------------+-----------------------------+
+| ``elasticsearch_socket_timeout``                   | Duration  | Timeout when sending/receiving from Elasticsearch connection | ``60s`` (60 Seconds)        |
++----------------------------------------------------+-----------+--------------------------------------------------------------+-----------------------------+
+| ``elasticsearch_version``                          | (2 or 5)  | Major version of Elasticsearch being used in the cluster     | ``5``                       |
++----------------------------------------------------+-----------+--------------------------------------------------------------+-----------------------------+
+| ``elasticsearch_discovery_enabled``                | boolean   | Enable automatic Elasticsearch node discovery                | ``false``                   |
++----------------------------------------------------+-----------+--------------------------------------------------------------+-----------------------------+
+| ``elasticsearch_discovery_filter``                 | String    | Filter by node attributes for the discovered nodes           | empty (use all nodes)       |
++----------------------------------------------------+-----------+--------------------------------------------------------------+-----------------------------+
+| ``elasticsearch_discovery_frequency``              | Duration  | Frequency of the Elasticsearch node discovery                | ``30s`` (30 Seconds)        |
++----------------------------------------------------+-----------+--------------------------------------------------------------+-----------------------------+
 
-  # Public IP address or host name of the Graylog node, accessible for the other Elasticsearch nodes
-  elasticsearch_network_host = 198.51.100.23
+Automatic node discovery
+^^^^^^^^^^^^^^^^^^^^^^^^
 
+.. caution:: Automatic node discovery will be available starting with Graylog 2.3.0-alpha.3/beta.1.
```

Comment:
The point why I put it in, is that there are users who want to use the documentation to get `alpha.2` up and running. If they read about automatic node discovery, it could confuse them when it's just not there for their version. I will leave it in and we will remove it when the next alpha/beta is out.